release/release.go

```diff
@@ -622,6 +681,43 @@ func LatestRC(ctx context.Context, owner, repo, k8sVersion string, client *githu
 
 }
 
+// rke2ChartVersion will return the version of the rke2 chart from the chart versions file
+func rke2ChartsVersion(branchVersion string) (map[string]chart, error) {
+
+	chartVersionsURL := "https://raw.githubusercontent.com/rancher/rke2/" + branchVersion + "/charts/" + rke2ChartsVersionsFile
```

Comment:
Can this be a const?

Reply:
this can't be a constant unfortunately because it depends on the milestone version being passed to the function